moduleroot/.github/settings.yml.erb

```
@@ -0,0 +1,52 @@
+<% if @configs['repository'] -%>
+repository:
+<%   @configs['repository'].each do |key, value| -%>
+  <%= key %>: <%= value %>
+<%   end -%>
+<% end -%>
+<% if @configs['labels'] -%>
+
+labels:
+<%   @configs['labels'].each do |label| -%>
+  - name: <%= label['name'] %>
+<%     label.keys.each do |k| -%>
+<%       next if k == 'name' -%>
+    <%= k %>: <%= label[k] %>
+<%     end -%>
+<%   end -%>
```

Not sure if delete is pretty, but I think it works:
```suggestion
  - name: <%= label.delete('name') %>
<%     label.each do |key, value| -%>
    <%= key %>: <%= value %>
<%     end -%>
<%   end -%>
```